packages/store/src/releasesSlice/thunks.ts

```diff
@@ -154,7 +171,7 @@ export const createArtifact = createAsyncThunk(`${sliceName}/createArtifact`, ({
     GeneralApi.upload(
       `${deploymentsApiUrl}/artifacts/generate`,
       formData,
-      e => dispatch(uploadProgress({ id: uploadId, progress: progress(e) })),
+      (e: { loaded: number; total: number }) => dispatch(uploadProgress({ id: uploadId, progress: progress(e) })),
```

Comment:
maybe `e: AxiosProgressEvent`?

Reply:
Hmm indeed the proper type for `total` - although I guess in our case the files are clearly defined... I guess to align with that type `progress()` should fall back to the `loaded` value, but OTOH in this case I'd rather stick to your type for now and leave the axios alignment for later... 👌